### DISCUSSION TITLE
Is there a way to make it such that the particles come out of all corners and not just one?

### DISCUSSION THREAD

**Discussion by IlyaTaidi** [matteobruni/tsparticles#1832] [Q&A] [answered]
All my issues have been fixed by changing the way that settings are loaded

**Comment by matteobruni** [comment, marked as answer]
I was trying your config for the links flickering and the particles appears correctly inside the canvas and not in a corner or from the corners.

Do you have some other code that refreshes the container or resizes the canvas? Which component are you using? React?

**Reply by IlyaTaidi** [reply]
Negative on anything that refreshes the container. I'm Using Ionic/Angular. Extremely barebones, only html element is the ngparticles container, and the only thing in the ts file is the options.

I had tried to download the demo and even in that setup where it was using the container the same issues were appearing.

As of right now the flickering is gone now that the particles bounce on the edges, but the particles all come up out of that bottom corner and autoplay is still not working.

**Reply by IlyaTaidi** [reply]
It has something to do with the move subsection of the options. 
Using these move options, disables the flicker but makes the particles appear from bottom right

      move: {
        direction: 'none',
        enable: true,
        outMode: 'bounce',
        random: true,
        speed: 1.5,
        straight: false
      },

Using these move options, makes the particles appear in the right space, but flicker as they try to connect to particles off screen

      "move": {
        "attract": {
          "enable": false,
          "rotate": {
            "x": 600,
            "y": 1200
          }
        },
        "direction": "none",
        "distance": 0,
        "enable": true,
        "gravity": {
          "acceleration": 9.81,
          "enable": false,
          "maxSpeed": 50
        },
        "noise": {
          "delay": {
            "random": {
              "enable": false,
              "minimumValue": 0
            },
            "value": 0
          },
          "enable": false
        },
        "random": false,
        "size": false,
        "speed": 0.6,
        "straight": false,
        "trail": {
          "enable": false,
          "length": 10,
          "fillColor": {
            "value": "#000000"
          }
        },
        "vibrate": false,
        "warp": false
      },

**Reply by IlyaTaidi** [reply]
Simply adding (outMode: 'bounce',) to the latter move section makes it behave as the former, changing the spawn location of the particles
